chargebee_byte/parameters.py

```diff
@@ -0,0 +1,22 @@
+def generate_parameters(parameters, operators):
+    return ['{}[{}]'.format(parameter, operator) for parameter in parameters for operator in operators]
```

Comment:
I really dislike nested comprehensions, I find them completely unreadable, but I think it's acceptable here. It's a simple function and we have it under test anyway

Reply:
I completely agree, but this is just about the one case I think it's a little readable. But yeah I personally always go for fully written out loops. But then I would always here "you can make this a list comprehension" so yeah... :p

Reply:
You could always fight back on that, I would never agree to changing code into a nested list comprehension :D but like I said for this one it's fine, leave it

Reply:
Well, sometimes I do think it's nice, but not always.